heka/meta/heka.yml

```diff
@@ -336,7 +336,8 @@ aggregator:
           threshold: 50
           function: percent
     - status: okay
-    # A policy which is typically used for stateless clusters
+    # A policy that is used to derive a cluster status based
+    # on the status okay or down status for its members.
```

Comment:
Also for "simple" GSE definition like [GlusterFS](https://github.com/tcpcloud/salt-formula-glusterfs/blob/master/glusterfs/meta/heka.yml#L76), we need to introduce intermediate GSE definition to avoid confusion in the future:

```
    glusterfs_service:
      policy: availability_of_members
      alerting: enabled
      match:
        member: glusterfs_check
      group_by: hostname
      members:
      - glusterfs_check
      dimension:
        service: glusterfs
        nagios_host: 01-service-clusters
    glusterfs:
      policy: highest_severity
      alerting: enabled_with_notification
      match:
        service: glusterfs
      members:
      - glusterfs_service
      dimension:
        cluster_name: glusterfs
        nagios_host: 00-top-clusters
```